source/PlayerInfo.h

```diff
@@ -377,6 +392,11 @@ class PlayerInfo {
 	std::vector<const System *> travelPlan;
 	const Planet *travelDestination = nullptr;
 
+	RelocateStatus relocationStatus = RelocateStatus::NONE;
+	bool relocateFlagshipOnly = false;
+	const Planet *relocationPlanet = nullptr;
+	const Planet *oldRelocationPlanet = nullptr;
+
```

Comment:
Can this go in a `Relocate` (or `Relocation`, or some other name) class instances of which will store the parameters of the relocation and handle the specific methods needed to do it? This is going to get rather complicated once support for, for example, only moving ships in specific systems, for example, is an option.

Reply:
Yes, maybe a bit early to already do this now (for just 4 variables), but we should definitely do this when we get more complex Relocation options. (And I don't mind to already have this done now in this PR.)

Reply:
I will see how I will do it, the class could hold some functions too

Reply:
created just a class for now